src/async_jobs.ts

```diff
@@ -92,6 +95,14 @@ async function slackInteractivityHandler(
         logger.info(
           `SERVER POST /slack-interactivity: Determined user interaction is a message shortcut.`
         );
+
+        const { viewId } = interactivityMetadata;
```

Comment:
Why is this necessary if all it's carrying is a viewId (ie why don't you just pass the viewId)? In case we want to carry more than the viewId in the future?

Reply:
Yup, that was my thinking.